packages/page-accounts/src/types.ts

```diff
@@ -20,8 +21,15 @@ export interface ModalProps {
   onStatusChange: (status: ActionStatus) => void;
 }
 
+export interface Delegation {
+  accountDelegated: string
+  amount: Balance
+  conviction: Conviction
+}
+
 export interface SortedAccount {
   account: KeyringAddress;
   children: SortedAccount[];
   isFavorite: boolean;
+  delegation?: Delegation;
```

Comment:
Just move up a line. (Same as above, the normal eslint rules for sorting doesn't check interfaces... not sure as to "why")